config/configmaps/install-log-regexes-configmap.yaml

```
@@ -120,6 +120,11 @@ data:
       - "current credentials insufficient for performing cluster installation"
       installFailingReason: AWSInsufficientPermissions
       installFailingMessage: AWS credentials are insufficient for performing cluster installation
+    - name: AWSThrottlingIssues
+      searchRegexStrings:
+      - "error waiting for Route53 Hosted Zone"
+      installFailingReason: AWSThrottlingIssues
+      installFailingMessage: AWS is having throttling issues, please retry your installation
```

I'd move this to line 86 (after Generic Throttling) and reuse the same Reason

```suggestion
    - name: AWSRoute53Throttling
      searchRegexStrings:
      - "error waiting for Route53 Hosted Zone .* creation: timeout while waiting for state to become 'INSYNC'"
      installFailingReason: AWSAPIRateLimitExceeded
      installFailingMessage: AWS API rate limit exceeded waiting for Route53
```

---

i made the regexp a little more aggressive too